ee/control/consumers/remoterestartconsumer/errors.go

```diff
@@ -0,0 +1,35 @@
+package remoterestartconsumer
+
+import (
+	"errors"
+	"fmt"
+)
+
+// ErrRemoteRestartRequested is returned to the main launcher rungroup when
+// a remote restart has been requested.
+type ErrRemoteRestartRequested struct {
```

Comment:
nit, I think the typical convention is to have `Err` at the end of the struct name?

```suggestion
type RemoteRestartRequestedErr struct {
```

Reply:
Ah, I was confusing the naming convention for struct errors with the naming convention for sentinel errors like `os.ErrNotExist` and `http.ErrServerClosed`. Per your other comment I'll update to use a sentinel error.

Reply:
that's a good point, not sure what "correct" is =)

Reply:
The way you suggested is more concise and much easier to read!